linkerd/docs/protocol-h2.md

```diff
@@ -71,6 +71,11 @@ When TLS is not configured, h2 servers accept both
 requests.  Plaintext clients are currently only capable of issuing
 prior-knowledge requests.
 
+
+<aside class="warning">
+Note that HTTP/2 clients should be careful not to send any DATA frame whose body exceeds 1/4th of the stream window size, as doing so may cause the connection to hang.
```

Comment:
* Technically, frames larger than `WINDOW_SIZE - CLASSIFIED_BUFFER_SIZE` can hang.  So if, for example, someone were to set the initial window on a stream to 8k, they're going to have a bad time.
* I believe the stream will hang, not the connection.